wmde/FundraisingFrontend
# T172884 Move test templates to dedicated skin

* document the idea of skins
* mv `/tests/templates` to `/skins/test/templates`
* remove actual application templates (`/app/templates`) from paths used as template source during tests
* create minimal implementations of all files loaded during (EdgeToEdge) test run by the application (previously loaded from `/app/templates`)
  * I reduced the complexity of these by a lot, yet let them stay somewhat sane HTML, not pure key-value-pairs. Either way, this remains/adds to the known flaw - the EdgeToEdge assert against this, instead of the actual 10h16 skin.
* **duplicate Mail_\* templates** in the test template folder until #946 is ready. They can probably simply be deleted thereafter.

Make sure you do not have a `config.test.local.json` interfering with the change to twig.loaders.filesystem.

For the record: There are technically less problems (13 Errors, 18 Failures) pointing the EdgeToEdge tests at (only) the actual 10h16 templates, than there where pointing them at only the test templates. (Not saying these would be quicker/easier to fix)
